### PR TITLE
Update attestation service deployment docs

### PR DESCRIPTION
**Pull request by eelanagaraj**
Stacked PR for attestation service deployment docs related to keystore management changes PR #8257

Edit: tested that the new process works as expected when running the recently built attestation service (from the above PR)

### CODE REVIEW

File: packages/docs/validator-guide/attestation-service.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ This guide steps you through setting up an Attestation Service:
 * Follow the instructions to set up a validator on [mainnet](../getting-started/running-a-validator-in-mainnet.md) or [baklava](../getting-started/running-a-validator-in-baklava.md)
 * Configure Twilio, MessageBird and Nexmo, the SMS providers used by Attestation Service
 * Generate and register an attestation signer key
-* Deploy a Celo full node, with the attestation signer key unlocked
+* (for versions prior to 1.3.0) Deploy a Celo full node, with the attestation signer key unlocked
 * Deploy the attestation service
 * Configure and publish validator metadata so that clients can find your attestation service
-* Configure monitoring for the full node and attestation service
+* Configure monitoring for the attestation service and if applicable the full node
```
Comment (codyborn) [inline packages/docs/validator-guide/attestation-service.md]: Totally optional - might make it more clear
```suggestion
* Configure monitoring for the attestation service and the full node (if applicable)
```

File: packages/docs/validator-guide/attestation-service.md [hunk 2]
```diff
@@ -33,9 +33,13 @@ The `PORT` environment variable sets the listening port for the service on the l
...
+
+While it is still possible to use the full node method described above, from version 1.3.0 onwards, it is recommended to instead deploy the Attestation Service with access to a keystore file that encrypts the attestation signer key. This allows the service to sign attestations locally and then submit these to the proxy node, instead of depending on the separate full node instance to both sign and submit attestations. The passphrase for decrypting the keystore file must be passed in as an environment variable when starting up the Attestation Service. The new method of signing is expected to reduce Attestation Service errors due to the full node not being synced and generally reduce operating overhead.
+
+Attestation Service is a stateless service that uses a database to persist status of current and recently completed SMS delivery attempts. The most straightforward deployment architecture is to have a single machine or VM running three containers (two for versions 1.3.0 onwards): one the attestation service, a Celo Blockchain node (only for versions prior to 1.3.0), and a single database instance.
```
Comment (codyborn) [inline packages/docs/validator-guide/attestation-service.md]: Nit: "The Attestation Service ..."

**Review by codyborn — APPROVED** [review]
LGTM!

File: packages/docs/validator-guide/attestation-service.md [hunk 3]
```diff
@@ -33,9 +33,13 @@ The `PORT` environment variable sets the listening port for the service on the l
...
+From Attestation Service version 1.3.0 onwards, there are two ways of deploying the Attestation Service. We recommend that new validators use the file keystore method (instead of using a Celo full node) for key management, and that existing validators eventually migrate to the new architecture.
 
-Attestation Service is a stateless service that uses a database to persist status of current and recently completed SMS delivery attempts. The most straightforward deployment architecture is to have a single machine or VM running three containers: one the attestation service, a Celo Blockchain node, and a single database instance.
+An Attestation Service prior to version 1.3.0 is usually deployed alongside a Celo full node instance, which needs to have the attestation signer key unlocked. This can be either deployed on the same physical machine, or in a VM or container on a different host. It is possible but not recommended to use a proxy node as the associated full node, but in this case ensure RPC access is locked down only to the Attestation Service.
```
Comment (aslawson) [inline packages/docs/validator-guide/attestation-service.md]: >  It is possible but not recommended to use a proxy node as the associated full node

I'm assuming this refers to enabling signing by the Validator proxy nodes?

File: packages/docs/validator-guide/attestation-service.md [hunk 4]
```diff
@@ -244,18 +265,46 @@ celocli account:get-metadata $CELO_VALIDATOR_RG_ADDRESS
...
+
+### (recommended) Running the Attestation Service v1.3.0+
+
+The main difference between the old method of running the Attestation Service alongside a Celo full node is that you will need to ensure that Docker can access the directory containing the `keystore` which stores the `CELO_ATTESTATION_SIGNER_ADDRESS`'s private key as an encrypted file.
+
+To do this, you need to map the directory containing the `keystore` to the Attestation Service's Docker volume and set the environment variable `ATTESTATION_SIGNER_KEYSTORE_DIRPATH` to the path to this directory relative to the Docker container. You will also need to set the environment variable `ATTESTATION_SIGNER_KEYSTORE_PASSPHRASE` to the password used during the creation of the `CELO_ATTESTATION_SIGNER_ADDRESS`.
+
+Additionally, ensure that the `CELO_PROVIDER` environment variable points to a Celo node that can be used to instantiate ContractKit; this can be a separate full node (either one you run or forno) or, depending on your configuration, your validator proxy node.
```
Comment (aslawson) [inline packages/docs/validator-guide/attestation-service.md]: We should not be recommending forno as primary use for production services.

Instead, we can refer to other node service providers. Ultimately, we should have a docs page with all the node-as-a-service options, but for now we can say:
`either one you run or an endpoint to a node service provider like Figment Datahub`

Ideally, we'd want to recommend leveraging the validator proxy node for reading chain data, but if there are limitations that make this unfeasible for most, let's chat more about that.

Comment (eelanagaraj) [reply packages/docs/validator-guide/attestation-service.md]: Note: discussed on slack -- added a note about Figment now; will add further docs changes regarding backup nodes in a later PR.

**Pull request merged**